app/views/transition_landing_page/_buckets.html.erb

```diff
@@ -3,7 +3,7 @@
 <div class="landing-page__section"
   data-analytics-ecommerce
   data-ecommerce-start-index="1"
-  data-list-title="Brexit landing page: <%= guidance_header %>"
+  data-list-title="<%= t("transition_landing_page.data_list_title", header: guidance_header) %>"
```

Comment:
Thanks for making this change.

Do you know how Rails will handle the four double quotes?  Maybe the two around `transition_landing_page.data_list_title` should be single quotes?

Reply:
I don't think it we need to use single quotes as Rails seems to render the attribute with double quotes correctly. 

Below is a little snippet from the debugger output after running the transition landing page feature:

```
<div class=\"landing-page__buckets\">\n    \n<div class=\"landing-page__section\" data-analytics-ecommerce data-ecommerce-start-index=\"1\" data-list-title=\"Brexit landing page: Changes for businesses and citizens\" data-search-query>
```

Reply:
Brillaint!  Thanks for the clarification.

Reply:
No worries, thanks for checking!